ci-operator/config/openshift/grafana-tempo-operator/openshift-grafana-tempo-operator-main.yaml

```diff
@@ -25,14 +33,17 @@ operator:
   substitutions:
   - pullspec: ghcr.io/grafana/tempo-operator/tempo-operator:v0.2.0
     with: pipeline:tempo-operator
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
```

Comment:
```suggestion
      version: "4.14"
```

Reply:
For presubmits, we have kept the version to 4.13 as there are issues with claiming 4.14 clusters.